lab/index.html

```diff
@@ -3,55 +3,79 @@
   <header>
   <title>Delicious</title>
   <link href="css/style.css" type="text/css" rel="stylesheet"/>
+  <link href="https://fonts.googleapis.com/css?family=Domine|Lobster|Philosopher" rel="stylesheet">
   </header>
   <head>
     <div class="row">
     <div class="column col-1">
-  <img id ="logo" src="lab-assets/logo.png"/>
+      <img id ="logo" src="lab-assets/logo.png"/>
+      </div>
+      <div class ="column col-3">
+      <h1 id="delicious">Delicious</h1>
+      <h6>THE BEST FOOD BLOG ON THE WEB</h6>
+      </div>
+      <div id="right" class="column col-8">
+      <img src="lab-assets/fb-icon.png"/>
+      <img src="lab-assets/twit-icon.png"/>
+      <img src="lab-assets/gp-icon.png"/>
+      <img src="lab-assets/flic-icon.png"/>
+      <img src="lab-assets/insta-icon.png"/>
+      <img src="lab-assets/pint-icon.png"/>
+      <img src="lab-assets/rss-icon.png"/>
+      <img src="lab-assets/mail-icon.png"/>
+      </div>
 </div>
 <div class= "row">
 <div class="column col-12">
 <img id ="border" src="lab-assets/border.png"/>
 </div>
 </div>
-<div class ="column col-3">
-<h1>Delicious</h1>
-<h6>THE BEST FOOD BLOG ON THE WEB</h6>
-</div>
-<div id="right" class="column col-6">
-<img src="lab-assets/fb-icon.png"/>
-<img src="lab-assets/twit-icon.png"/>
-<img src="lab-assets/gp-icon.png"/>
-<img src="lab-assets/flic-icon.png"/>
-<img src="lab-assets/insta-icon.png"/>
-<img src="lab-assets/pint-icon.png"/>
-<img src="lab-assets/rss-icon.png"/>
-<img src="lab-assets/mail-icon.png"/>
-</div>
 </div>
 </head>
 <body>
   <div class ="row">
     <div class="column col-6">
 <h1 id="title">Chocolate Pizza</h1>
```

Comment:
Your indentation throughout your HTML is off. For example this tag, since it is inside a div, should be spaced inward two more spaces(1 tab).